src/k-provider/ovp/loaders/bumper-entry-loader.js

```diff
@@ -0,0 +1,86 @@
+//@flow
+import RequestBuilder from '../../../util/request-builder';
+import OVPBaseEntryService from '../services/base-entry-service';
+import OVPConfiguration from '../config';
+import KalturaPlaybackContext from '../response-types/kaltura-playback-context';
+import KalturaMediaEntry from '../response-types/kaltura-media-entry';
+import KalturaBaseEntryListResponse from '../response-types/kaltura-base-entry-list-response';
+import Error from '../../../util/error/error';
+
+type OVPBumperEntryLoaderResponse = {
+  entry: KalturaMediaEntry,
+  playBackContextResult: KalturaPlaybackContext
+};
+export type {OVPBumperEntryLoaderResponse};
+
+export default class OVPBumperEntryLoader implements ILoader {
+  _entryId: string;
+  _requests: Array<RequestBuilder>;
+  _response: ?OVPBumperEntryLoaderResponse = null;
+
+  static get id(): string {
+    return 'bumper';
+  }
+
+  /**
+   * @constructor
+   * @param {Object} params loader params
+   */
+  constructor(params: Object) {
+    this.requests = this.buildRequests(params);
+    this._entryId = params.entryId;
+  }
+
+  set requests(requests: Array<RequestBuilder>) {
+    this._requests = requests;
+  }
+
+  get requests(): Array<RequestBuilder> {
+    return this._requests;
+  }
+
+  set response(response: any) {
+    if (response[0].hasError) {
```

Comment:
handling error here seems to break or way of handling this in other loaders, and especially creating an error here.